src/approxmc.cpp

```diff
@@ -199,6 +199,11 @@ DLL_PUBLIC bool AppMC::get_reuse_models()
     return data->conf.reuse_models;
 }
 
+DLL_PUBLIC void AppMC::setup_vars()
```

Comment:
You declare and define this function but you don't use it, right? Can we remove it?

Reply:
Glad to hear you like the code! It's my first time writing Python bindings or any decent chunk of C++ so definitely let me know if anything looks funky. This functions was for the Unigen bindings I wrote (https://github.com/meelgroup/unigen/pull/7) to setup the internal solver for Unigen without having to count everything, but please let me know if there's a better way to do that! I'll go into more detail on that PR.

Reply:
Ahha, I see! Merging now :)